.changes/next-release/bugfix-NettyNIOHTTPClient-dbb46f2.json

```diff
@@ -0,0 +1,6 @@
+{
+    "category": "Netty NIO HTTP Client", 
+    "contributor": "", 
+    "type": "bugfix", 
+    "description": "Fixed the issue where BAD_CERTIFICATE issue manifested as acquire connection timeout error when using TLS1.3 and proxy."
```

Comment:
nit: is this issue unique to `BAD_CERTIFICATE`?

Reply:
Probably not, I'll update it